src/sync.js

```diff
@@ -1,31 +1,27 @@
+
 import { observe } from 'mobx';
 
 export const syncHistoryWithStore = (history, store) => {
   // Initialise store
   store.history = history;
 
   // Handle update from history object
-  const handleLocationChange = (location) => {
-    store._updateLocation(location);
-  };
+  const handleLocationChange = store._updateLocation;
```

Comment:
This change will break the `_updateLocation` call, as the `this` context will not be bound correctly. Might just be easier to revert this change.